src/RestfulClient.cs

```diff
@@ -69,67 +70,31 @@ public RestfulClient(string apiEndpoint, HttpMessageHandler messageHandler, para
 
         #region Sync Methods
 
-        /// <summary>
```

Comment:
I have done this in my PR but you can keep it as you are changing the interface.